lib/container_orchestrator/object_definition.rb

```diff
@@ -1,3 +1,5 @@
+require 'jwt'
```

Comment:
Please move this require inside of the ` def opentofu_runner_token` method since that is the only caller.  Only the orchestrator should call that method, so other workers will not load it.